ndsl/quantity.py

```diff
@@ -298,7 +300,7 @@ def __init__(
         """
         # ToDo: [Florian 01/23] Kill the abomination.
         # See https://github.com/NOAA-GFDL/pace/issues/3
-        from ndsl.dsl.typing import Float
+        # from ndsl.dsl.typing import Float
```

Comment:
I'd rather delete this instead of commenting it out

Reply:
I have tested these changes in the following ways:

- Test branch in pyFV3
    - python=3.8 
        - translate numpy passes
        - translate dace:cpu passes
    - python=3.11
        - translate numpy passes
        - translate dace:cpu passes
- Test branch in pySHiELD
    - python=3.8
        - translate numpy passes
        - translate dace:cpu passes
    - python=3.11
        - translate numpy passes
        - translate dace:cpu passes
- Test branch in pace
    - python=3.8
        - tests/main passed
    - python=3.11
        - tests/main passed